x11-wm/awesome/awesome-4.3-r2.ebuild

```diff
@@ -76,16 +82,11 @@ src_configure() {
 		mycmakeargs+=("-DLUA_INCLUDE_DIR=${EPREFIX}/usr/include/luajit-2.0")
 		mycmakeargs+=("-DLUA_LIBRARY=${EPREFIX}/usr/$(get_libdir)/libluajit-5.1.so")
 	fi
-	cmake-utils_src_configure
-}
-
-src_test() {
```

Comment:
did you run tests?

Reply:
I tried, but tests fail. They are already restricted, I just added a bug reference and restored `src_test` if someone wants to fix it in the future.